subjects/regressionTests.py

```diff
@@ -81,12 +81,17 @@ def cmdline_split(s):
         a.append(sub)
     return a
 
+# On Windows file removal is asynchronous but renames are atomic
```

Comment:
...and on Linux, this amounts to the same thing -- `rename` is a link / unlink, followed by a finally a link?

Reply:
No, on Linux the file entry is removed at unlink(), but the data is release on last close. After unlink the file is gone, except if you have an open handle still....

Reply:
and rename is an actual syscall, so not link/unlink, but a specific function registered in the VFS handler